tests/unit/components/estimated-cost-test.js

```diff
@@ -3,6 +3,18 @@ import {
   test
 } from 'ember-qunit';
 
+var stubFeature = function(component, feature, value) {
+  component.features = {
```

Comment:
@mixonic Seems like feature stubbing is built into acceptance tests, but not unit tests.  Is there a better way to do this?